pkgs/top-level/aliases.nix

```diff
@@ -412,12 +412,19 @@ mapAliases {
   elixir_ls = elixir-ls; # Added 2023-03-20
 
   # Emacs
+  emacs28 = throw "Emacs 28 is removed due to CVEs which are fixed in Emacs 30"; # Added 2025-03-03
   emacs28-gtk2 = throw "emacs28-gtk2 was removed because GTK2 is EOL; migrate to emacs28{,-gtk3,-nox} or to more recent versions of Emacs."; # Added 2024-09-20
```

Comment:
Nit: reword this message so that no one gets it wrong by accident.

Reply:
Which part may be confusing?  Would it be better to remove `which are fixed in Emacs 30`?

Reply:
```nix
  emacs28-gtk2 = throw "Emacs 28 is removed due to CVEs which are fixed in Emacs 30"; # Added 2024-09-20
```

Supposing someone did not update their system in a long heck of time, the "update to emacs29 or use other toolkit" message can be misleading.

Reply:
Oh, you are referring to the emacs28-gtk2 alias which is not touched by this PR.

If a user of that emacs28-gtk2 alias switches to emacs28 or emacs28-gtk3, he will follow instructions added by this PR to use emacs 30.  Yes, it is not great for a user to have to change attr twice, but I do not think it is confusing/misleading.